### PR TITLE
fix: multiple ajax

### PR DESCRIPTION
**Pull request by tiger2005**

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -20,17 +20,24 @@ const getInfo = (id: number): UserInfo => {
...
+  useEffect(() => {
+    const userInfo = getInfo(uid);
+    if (userInfo === undefined) {
+      console.error(`Get user ${uid}'s info failed!`);
+      getInfoOK = false;
+    }
+    setData(userInfo);
+    setLoadType(true);
+  }, [props.requestID]);
+  return loaded ? (getInfoOK ? <InfoCard {...data} /> : <FailedCard />) : <div></div>;
```
Comment (Nikaidou-Shinku) [inline src/components/Card.tsx]: empty div use `<div />`.

**Review by Nikaidou-Shinku — CHANGES_REQUESTED** [review]

File: src/components/Card.tsx (hunk 1 above)
Comment (tiger2005) [reply src/components/Card.tsx]: Fixed.

File: src/styles/cardStyles.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const CARD_STYLE: CSSProperties = {
   textAlign: "center",
   textSizeAdjust: "100%",
   lineHeight: "25.6px",
-  textDecoration: "none"
+  textDecoration: "none",
```
Comment (Nikaidou-Shinku) [inline src/styles/cardStyles.ts]: Omit the comma of the last item of an object.

**Review by Nikaidou-Shinku — CHANGES_REQUESTED** [review]

File: src/components/CardLoader.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ const getCardStyle = (pos: { x: number, y: number }) => {
     zIndex: floatNumber,
     opacity: 0,
     top: 0,
-    left: 0
+    left: 0,
```
Comment (Nikaidou-Shinku) [inline src/components/CardLoader.tsx]: Omit the comma of the last item of an object.

**Review by Nikaidou-Shinku — CHANGES_REQUESTED** [review]

File: src/styles/cardStyles.ts (hunk 2 above)
Comment (tiger2005) [reply src/styles/cardStyles.ts]: Done.

File: src/components/CardLoader.tsx (hunk 3 above)
Comment (tiger2005) [reply src/components/CardLoader.tsx]: Done.

**Review by Nikaidou-Shinku — APPROVED** [review]

**Pull request merged**